### DISCUSSION TITLE
Use Control Strip concept as taskbar for hello os

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#394] [Ideas]
![Mac_OS_7 6 1_emulated_inside_of_SheepShaver](https://user-images.githubusercontent.com/44593430/200357132-11683417-e36e-47bd-9a43-40deeafabbd5.png)
The one on the bottom left.
![image-asset (1)](https://user-images.githubusercontent.com/44593430/200357464-c7bc8240-4d68-4a7d-ae7a-bbc678ff2e98.png)
Control Strip

If we use the concept of Control Strip of Mac OS 7 as taskbar, we can use more windows, it can shrink to a little button, and can press the button or mouse idle to pull out the taskbar to switch windows.

**Comment by probonopd** [comment]
Thanks for your suggestion but this one is very unlikely to be considerd for implementation (at least by me) as the Control Strip was bolted onto System 7 only around the introduction of the PowerBooks. it is not an integral part of the desktop metaphor and was removed again later.
